packages/user-mgnt/src/config/routes.ts

```diff
@@ -559,6 +561,16 @@ export const getRoutes = () => {
         }
       }
     },
+    {
+      method: 'GET',
+      path: '/getAllSystems',
```

Comment:
@favour234 is this route still used?  If not we should delete it, and also delete the getAllSystemsHandler function.  I dont see the code in this PR that is using it.